src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  margin-top: 20px;
+}
+
+.nav-list-el:not(:first-child) {
+  padding: 0;
+  margin-left: 20px;
+}
+
+.nav-link {
+  text-transform: uppercase;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  color: black;
+  padding-top: 23px;
+  padding-bottom: 22px;
+}
+
+.logo {
+  margin: 10px 0 10px 50px ;
+}
```

Comment:
Dont use margins for positioning elements. Remove margin. Add padding to `header` instead